pamelalozano/src/main/webapp/index.html

```diff
@@ -125,6 +128,76 @@ <h3>A little taste of my life:</h3>
 </div>
 </div>
     
+<div class="container" id="skills">
```

Comment:
I though I had a commit of this section, my mistake. 
I added the skills section from a template.